src/lib/gitlab.ts

```diff
@@ -69,99 +98,21 @@ export class Gitlab {
         let gitlabCommits: ICommitData[] = [];
         const repoNameId = url.slice(19).split("/").join("%2F");
         const urlL = global_APIURLs.gitlab + "projects/" + repoNameId + "/repository/commits?all=1";
-        if (isPrivate) {
-            await fetch(urlL, {
-                headers: this.headers_gitlabRequestAuth
-            })
-            .then(res => { if (res.status == 200) return res.json() })
-            .then(async commits => {
-                await Promise.all(commits.map(async (data: any) => {
-                    let obj: ICommitData;
-                    let avatar = await this.getAvatar(data.author_email);
-                    let branchName = await this.getBranch(urlL.slice(0, -6) + "/" + data.id, true);
+        await fetch(urlL, {
+            headers: this.headers_gitlabRequestAuth
+        })
+        .then(res => { console.log(res.status); if (res.status == 200) return res.json() })
+        .then(async commits => {
+            await Promise.all(commits.map(async (commit: any) => {
+                let avatar = await this.getAvatar(commit.author_email);
+                let branchName = await this.getBranch(urlL.slice(0, -6) + "/" + commit.id, true);
+                let obj: ICommitData = makeCommitData(commit, branchName)
```

Comment:
Pass the avatar parameter to the commit data as we need it now `makeCommitData(commit, avatar, branchName)`